experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -311,4 +352,20 @@ export class PrometheusSerializer {
 
     return results;
   }
+
+  protected _serializeResource(resource: Resource): string {
+    const name = 'target_info';
+    const help = `# HELP ${name} Target metadata`;
+    const type = `# TYPE ${name} gauge`;
+
+    const resourceAttributes = filterReservedResourceAttributes(resource);
+
+    const otherAttributes = {
+      job: extractJobLabel(resource),
```

Comment:
If I read the spec text correctly, the SDK Prometheus exporters do not need to specialize the `job` and `instance` attributes. This is left to the Collector's Prometheus exporters.

Reply:
What happens when you don't use Collector's Prometheus exporter but let Prometheus directly scrape from `otel-js`? Where is this in the spec?

Reply:
The spec linked in the OP defines that resource attributes should be included in the info-typed metric — no words for the `job` and `instance` specialization.

> In SDK Prometheus (pull) exporters, resource attributes SHOULD be converted to a single [target_info metric](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#supporting-target-metadata-in-both-push-based-and-pull-based-systems); otherwise, they MUST be dropped, and MUST NOT be attached as labels to other metric families. The target_info metric MUST be an info-typed metric whose labels MUST include the resource attributes, and MUST NOT include any other labels. There MUST be at most one target_info metric exposed on an SDK Prometheus endpoint.

Reply:
@legendecas Yep this is correct. I must have missed that part about a dozen times when reading the spec there. Thanks for pointing this out :slightly_smiling_face: 
I removed this part again and now it's just serializing the resource without any modification :slightly_smiling_face: